packages/ingest/test/test-consumer.js

```diff
@@ -1,23 +1,30 @@
 'use strict';
 
 const delay = require('delay');
-const rewire = require('rewire');
+const proxyquire = require('proxyquire');
 const sinon = require('sinon');
 const test = require('ava');
 
-const consumer = rewire('../consumer');
-const Consumer = consumer.Consumer;
-
 const timeToReceiveMessages = 200; // ms
 const timeLimitModifier = 50;
 let testConsumer;
 
 async function stubReceiveSQSMessages(_url, { numOfMessages }) {
   await delay(timeToReceiveMessages);
-  return Array.apply(null, { length: numOfMessages }).map(() => 'i am a message'); // eslint-disable-line prefer-spread
+  return Array.apply(undefined, { length: numOfMessages }).map(() => 'i am a message'); // eslint-disable-line prefer-spread
 }
-consumer.__set__('receiveSQSMessages', stubReceiveSQSMessages);
-consumer.__set__('deleteSQSMessage', async () => true);
+
+const consumer = proxyquire(
```

Comment:
More of a preference thing, but could we achieve the same mocking effect by just doing:

```
const SQS = require('@cumulus/aws-client/SQS');
sinon.stub(SQS, 'deleteSQSMessage').resolves(true)
```

If so, is there any benefit to one approach vs the other?

Reply:
For consistency, 001345cae